charts/nfs-server-provisioner/templates/statefulset.yaml

```diff
@@ -117,6 +117,9 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.priorityClass.name }}
+      priorityClassName: {{ toYaml . }}
```

Comment:
Hmm, I guess one could use toYaml on a string. I had not considered that before. I'd do something like this typically -because it can be relevant to make it quoted if the name would be for example the string "5", but `toYaml .` would also do the trick.

```
      priorityClassName: "{{ . }}"
      priorityClassName: {{ . | quote }}
```

@joshuasimon-taulia I'm curious, have used this practice intentionally in favor of the practices I'm used to in this or other helm charts before?

Reply:
that was the result of copy/paste. i pushed an update